vyxal/helpers.py

```diff
@@ -43,3 +298,72 @@ def uncompress_str(string: str) -> str:
 def uncompress_num(num: str) -> int:
     # TODO (lyxal) Implement number (un)compression
     raise NotImplementedError()
+
+
+def vy_eval(item: str, ctx: context.Context) -> Any:
+    """Evaluates an item. Does so safely if using the online
+    interpreter"""
+
+    if ctx.online:
+        try:
+            return ast.literal_eval(item)
+        except:
+            # TODO: eval as vyxal
```

Comment:
Put in a `raise NotImplementedError` here?

Reply:
No, because it should return `item` if it isn't valid vyxal, meaning terminating isn't the most helpful behaviour.